packages/react-compass/src/menu-list/menu-list-dropdown.tsx

```diff
@@ -57,7 +60,11 @@ const MenuListDropdown = React.forwardRef<
         value={{isOpen: isControlled ? isOpenProp : isOpen, toggleOpen}}
       >
         {title}
-        {(isControlled ? isOpenProp : isOpen) && body}
+        <StyledMenuListDropdownBody
+          isOpen={(isControlled ? isOpenProp : isOpen) ? 'true' : 'false'}
```

Comment:
This should be a function that return the result. Check for using class name or different value instead of string true / false